Content.Server/Chat/Managers/ChatManager.cs

```diff
@@ -115,6 +115,10 @@ public void SendAdminAnnouncement(string message)
 
         public void SendHookOOC(string sender, string message)
         {
+            if (!_oocEnabled)
```

Comment:
CVar.
Some servers, albeit not us, may make the OOC channel read-only to the general population, in which case admins being able to use the relay to speak is good.